Day-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,14 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import './Button.css';
+
+function ActionButton(props) {
+    return (
+        <Fragment>
+            <div className={`actionButton ${props.buttonColor} ${props.buttonWidth}`} onClick={() => props.whenClicked(props.value)}>
+                {props.value}
```

Comment:
This seems like an opportunity for [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring). For example:
```javascript
function ActionButton({ buttonColor, buttonWidth, value, whenClicked }) {
  return (
    <div
      className={`actionButton ${buttonColor} ${buttonWidth}`} 
      onClick={() => whenClicked(value)}
    >
      {value}
    </div>
  );
}
```